cg/cli/workflow/nf_analysis.py

```diff
@@ -26,27 +21,12 @@
     show_default=True,
     help="Choose a configuration profile",
 )
-OPTION_TOWER = click.option(
-    "--with-tower",
-    is_flag=True,
-    default=False,
-    show_default=True,
-    help="Monitor workflow execution with Seqera Tower service",
-)
-OPTION_STUB = click.option(
```

Comment:
Won't we need this for development sometimes? Is there a way to trigger a stub run otherwise?

Reply:
I have used it in the beginning, mostly when I wanted to test file delivery. I am not sure it is relevant any more. On the pure pipeline development side, I don't think it is necessary